hoomd/hpmc/pytest/test_shape.py

```diff
@@ -24,14 +24,16 @@ def check_dict(shape_dict, args):
            and key != 'shapes':
             np.testing.assert_allclose(shape_dict[key], val)
         elif key == 'shapes':
-            for shape_args in shape_dict[key]:
+            for i in range(len(shape_dict[key])):
+                shape_args = shape_dict[key][i]
+                val_args = val[i]
                 for shape_key, shape_val in shape_args.items():
```

Comment:
Good change on the function in general we don't use `shape_val` now though so I suggest changing the line to this
```suggestion
                for shape_key in shape_args:
```